.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: Found something you weren't expecting?  Report it here!
+body:
+- type: markdown
+  attributes:
+    value: |
+      NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue.
```

Comment:
Should this still be here now that we have it already on the new issue landing page with the config.yaml?
Maybe an extra reminder here is not that bad.

Reply:
Too many reminders is never a bad thing if it stops security vulnerabilities being made public.
Ideally, it will make people reporting bugs think more than once, and that's if they even read far enough down.